README.md

```diff
@@ -22,17 +22,20 @@ Go to the official [Code For Life website][c4l].
 
 * Install prerequisites. E.g. on Ubuntu / Linux Mint:
     * `sudo apt-get install git`
-    * `sudo apt-get install python-dev`
+    * `sudo apt-get install python-dev python-pip`
```

Comment:
@wulmer experienced not having python-pip installed when he tried to follow the installation instructions. You are right that pip is usually installed, and I don't know what system he was using. We reasoned that listing the extra package would do no harm if already installed.